ansible/roles/compute_init/files/compute-init.yml

```diff
@@ -295,6 +297,12 @@
             cmd: "cvmfs_config setup"
       when: enable_eessi
 
+    - name: Configure VGPUs
+      include_role:
+        name: stackhpc.linux.vgpu
+        tasks_from: 'configure.yml'
```

Comment:
This appears to template udev rules, yet there are no calls to or handlers doing

udevadm control --reload-rules && udevadm trigger

Is this OK?

Reply:
So the udev rules are only used by the systemd units on startup, so I don't think we need to load them right away. It just for recreating everything at boot time.